### DISCUSSION TITLE
Admin client method getAdminEvents is missing resourceTypes query parameter

### DISCUSSION THREAD

**Discussion by jhollmannk** [keycloak/keycloak#8637] [Q&A] [answered]
We make heavy use of the Keycloak admin client library and have just discovered that the “resourceTypes” query parameter is missing in the definition of getAdminEvents in the RealmResource interface.
The REST endpoint “admin-events” does have that query parameter, it works just fine. We just cannot access it via the admin client method.
Is it OK to create a pull request for this minor change? I mean it's just adding
    @QueryParam(value="resourceTypes") List<String> resourceTypes
in the getAdminEvents method of the RealmResource interface.
I cannot get past the RedHat SSO to sign up for the issue tracker to create an issue for that (only get a termsSsoSignInMissingMessage "error message", don't know what happens there).

**Comment by jhollmannk** [comment]
Lost some characters due to markdown: it will be
`@QueryParam(value="resourceTypes") List<String> resourceTypes`
of course.

**Comment by stianst** [comment, marked as answer]
Hi, I see no issues with adding this to the admin client, and the admin client should be on-par with the admin rest api regardless, so a PR for this would be welcomed.
